.gitmodules

```diff
@@ -34,3 +34,10 @@
 	path = falcosecurity-libs
 	url = https://github.com/stackrox/falcosecurity-libs
 	branch = master
+[submodule "collector/proto/third_party/gogoproto"]
+	path = collector/proto/third_party/gogoproto
+	url = https://github.com/gogo/protobuf
+[submodule "collector/proto/third_party/stackrox"]
+	path = collector/proto/third_party/stackrox
+	url = https://github.com/stackrox/stackrox
+	branch = release/3.69.x
```

Comment:
Change to master.

Reply:
I changed to master for consistency with the other submodules.